.circleci/config.yml

```diff
@@ -409,32 +409,27 @@ jobs:
             tagname="AWS SDK for iOS "${CIRCLE_TAG}
             echo "$tagdescription" | github-release release -s ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -t ${CIRCLE_TAG} --name "$tagname" -d -
 
-  build_carthage:
+  build_xcframeworks:
     macos:
-      xcode: "12.1.0"
+      xcode: "12.3.0"
     steps:
       - skip_job_unless_required
       - checkout_with_cache
       - pre_start_simulator
       - update_brew
       - run:
-          name: Upgrade Carthage
-          command: |
-            HOMEBREW_NO_AUTO_UPDATE=1 brew upgrade carthage
-      - run:
-          name: Build Carthage
+          name: Build XCFrameworks
           command: |
-            # Build Carthage via this script to workaround lipo error due to multiple architectures in same file
-            bash ./CircleciScripts/carthage-build.sh --no-skip-current --platform iOS | tee buildout.txt
+            python3 ./CircleciScripts/create_xcframeworks.py
           no_output_timeout: 100m
       - save_cache:
-          key: carthage_build_result-{{ .Revision }}
+          key: build_xcframeworks-result-{{ .Revision }}
           paths:
-            - Carthage
+            - XCFrameworks
```

Comment:
feel free to downcase this; it was spelled `Carthage` before because that's what the tool expected